docs/src/main/sphinx/admin/properties-optimizer.md

```diff
@@ -52,11 +52,6 @@ aggregations or for mix of distinct and non-distinct aggregations.
 `AUTOMATIC` limits the use of `MarkDistinct` only for cases with limited
 concurrency (global or small cardinality aggregations), where direct distinct
 aggregation implementation cannot utilize CPU efficiently.
-`optimizer.mark-distinct-strategy` overrides, if set, the deprecated
```

Comment:
I assume the docs for the property itself either never existed or was just this info here.

Reply:
Yes, As far as I know, the docs were never there.